start.sh

```diff
@@ -15,6 +14,10 @@ if [ "$role" = "app" ]; then
 
     exec apache2-foreground
 
+elif [ "$role" = "queue" ] && [ "$HORIZON_ENABLED" = "1" ]; then
+
+    php /var/www/html/artisan horizon
```

Comment:
The [deploying horizon docs](https://laravel.com/docs/10.x/horizon#deploying-horizon) naturally talk about a more metal and less k8s/containerized environment.

I wonder if we need to do anything special here (for example they use `supervisor`). For the case of this terminating I think that it won't actually be necessary because I would expect the container in the pod to restart. Is that right?

I was also unclear if we need to handle some special cases when we are pushing new images. Is the code shared between pods in some way that necessitates us running this `php artisan horizon:terminate`? Or will creating the new pods with the new image be sufficient? Again I assume yes?

This is a long way to say that I expect us to need to do nothing and this is great but I wanted to ask.

Reply:
I would think :tm: all of the "supervisor" duties explained in the documentation are meaningless for us as we put the thing in a container and let k8s do the supervision for us.

> Or will creating the new pods with the new image be sufficient? Again I assume yes?

I would assume yes as well. I haven't noticed anything odd when rolling out new changes in local.